test/build.sh

```diff
@@ -138,7 +138,7 @@ function build-ci() {
   export REGISTRY="${NPD_STAGING_REGISTRY}/ci"
   export VERSION="$(get-version)-$(date +%Y%m%d.%H%M)"
   export TAG="${VERSION}"
-  make push
+  make push-tar build-container
```

Comment:
Maybe add a comment to explain that we do not need to push container here for CI tests.

Reply:
done